README.md

```diff
@@ -46,6 +46,9 @@ Additionally, some events will not be visible due to data retention time limits,
 
 Many of the these controls can be scoped down to the organizational unit level. We recommend [creating a new organization unit](https://support.google.com/a/answer/182537?hl=en#:~:text=An%20organizational%20unit%20is%20simply,level%20(parent)%20organizational%20unit) and applying these controls just to that new organizational unit for testing. Rerun ScubaGoggles after you've saved your configuration changes to see if the policy requirement is met.
 
+> [!IMPORTANT]
+> Use of this tool requires access to an internet browser to authenticate with Google Workspace as well as view the html report output.
```

Comment:
Slight tweak to clarify since consenting to the API scopes (and the other setup tasks within the GCP console) via the browser occur on the initial run but won't happen every time. Feel free to use or lose as you see fit.

```suggestion
> Use of this tool requires access to an internet browser for initial setup and to view html report output.
```

Reply:
Figure I'll just move it down one line to the top of the getting started section. My thought is that we want this warning up as high as possible so that if someone is using the guide to set up, they don't waste time with a few implementation steps before realizing they are not able to use the tool on their machine.